File: src/Domain/Entities/Book.cs
using Sharko.Domain.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sharko.Domain.Entities
{
    public class Book : BaseEntity
    {
        public string Name { get; set; }
        public string EngName { get; set; }
        public int Rating { get; set; }
        public string Image { get; set; }
        public string About { get; set; }

        public List<Tag> Tags { get; set; }
        public List<Category> Categories { get; set; }
        public List<ISBN> ISBNs { get; set; }
        public List<Author> Authors { get; set; }
        public List<Summary> Summaries { get; set; }
    }
}

File: src/WebUI/ClientApp/src/app/core/models/book/book.ts
import { AuthorInfo } from '../author/author-info';
import { Summary } from '../summary/summary';
import { SummaryInfo } from '../summary/summary-info';

export interface Book {
  id?: number;
  name?: string;
  rating?: number;
  image?: string;
  about?: string;
  tags?: string[];
  categories?: string[];
  isbns?: string[];
  authors?: AuthorInfo[];
  summaries?: SummaryInfo[];
}
